storage/migrations/README.md

```diff
@@ -6,10 +6,10 @@ each respective runtime.
 
 ## Naming Convention
 
-Migration filenames should follow the [convention](https://github.com/golang-migrate/migrate/blob/master/MIGRATIONS.md) `<id>_<name>.up.sql`, where `<id>` is the 4-digit index of the migration and `<name>` is a concise descriptor for the migration. For example, `0000_init.up.sql`.
+Migration filenames should follow the [convention](https://github.com/golang-migrate/migrate/blob/master/MIGRATIONS.md) `<id>_<name>.up.sql`, where `<id>` is the 2-digit index of the migration and `<name>` is a concise descriptor for the migration. For example, `00_init.up.sql`.
 
 We add the extra convention that migrations tied to major network upgrades specify `<name>` as `<chain_id>_init.sql`.
-For example, `0001_oasis_3_init.sql` might encode initialization for the [Damask upgrade](https://github.com/oasisprotocol/mainnet-artifacts/releases/tag/2022-04-11).
+For example, `01_oasis_3_init.sql` might encode initialization for the [Damask upgrade](https://github.com/oasisprotocol/mainnet-artifacts/releases/tag/2022-04-11).
```

Comment:
this sentence being no longer true of the damask migration, should we keep it as an example?

Reply:
To be honest, I missed the "_init" convention. I like the suggestion, let's simplify. Removed these two sentences.